Octokit.Tests/Clients/RepositoryCommentsClientTests.cs

```diff
@@ -240,6 +240,34 @@ public class TheDeleteMethod
         }
     }
 
+    public class TheReactionMethod
+    {
+        [Fact]
+        public void RequestsCorrectUrl()
+        {
+            NewCommitCommentReaction newCommitCommentReaction = new NewCommitCommentReaction(Reaction.Heart);
+
+            var connection = Substitute.For<IApiConnection>();
+            var client = new RepositoryCommentsClient(connection);
+
+            client.CreateReaction("fake", "repo", 1, newCommitCommentReaction);
+
+            connection.Received().Post<CommitCommentReaction>(Arg.Is<Uri>(u => u.ToString() == "repos/fake/repo/comments/1/reactions"), Arg.Any<object>(), AcceptHeaders.Reactions);
```

Comment:
You should hardcode the header string, so it serves as a sanity check against misspelling.